tox.ini

```diff
@@ -22,7 +22,7 @@ commands =
     "{envpython}" -m pytest -vvvv
 
 [testenv:devenv]
-envdir = {toxinidir}/venv
+envdir = {toxinidir}/.venv
```

Comment:
not exactly sure what this file is doing at this point.

we use poetry build now which i believe uses tox.